src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2269,6 +2269,8 @@ internal enum ErrorCode
 
         ERR_FeatureNotAvailableInVersion12 = 9202,
 
+        ERR_ExpectedInterpolatedString = 9203,
```

Comment:
Are there any special rules for new error codes? I just used the next available number

Reply:
Nope. Next available.

Reply:
I wonder how you resolve conflicts between different feature branches then since different branches might introduce different errors with same codes. It should be very painful to replace all comments in tests when merging changes